<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pic_label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOtw_rz"/>
    <addaction name="separator"/>
    <addaction name="actionKoniec"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Praca</string>
    </property>
    <widget class="QMenu" name="menuRodzaj_zadania">
     <property name="title">
      <string>Rodzaj zadania</string>
     </property>
     <addaction name="actionPrzeszukiwanie_wszerz"/>
     <addaction name="actionPrzeszukiwanie_wg_b"/>
    </widget>
    <addaction name="actionPraca_krokowa"/>
    <addaction name="actionRozwi_ca_e_zadanie"/>
    <addaction name="separator"/>
    <addaction name="menuRodzaj_zadania"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdycja"/>
  </widget>
  <action name="actionOtw_rz">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionKoniec">
   <property name="text">
    <string>Koniec</string>
   </property>
  </action>
  <action name="actionPraca_krokowa">
   <property name="text">
    <string>Praca krokowa</string>
   </property>
  </action>
  <action name="actionRozwi_ca_e_zadanie">
   <property name="text">
    <string>Rozwiąż całe zadanie</string>
   </property>
  </action>
  <action name="actionPrzeszukiwanie_wszerz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Przeszukiwanie wszerz</string>
   </property>
  </action>
  <action name="actionPrzeszukiwanie_wg_b">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Przeszukiwanie wgłąb</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
